benchmarks/Bench.cs

```diff
@@ -0,0 +1,81 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Benchmarks
+{
+    [Config(typeof(ConfigWithCustomEnvVars))]
+    public unsafe class AllocatorBenchmarks
+    {
+        private class ConfigWithCustomEnvVars : ManualConfig
+        {
+            private const string TiredCompilation = "COMPlus_TieredCompilation";
```

Comment:
Setting TieredCompilation shouldn't be required, BDN should run enough iterations to cause rejit automatically

Reply:
Right, you just asked for it, so I added. I can remove it

Reply:
Right, that was just to see if it actually made a difference. I think we concluded "yes to disassembly" and "no to perf numbers", so we probably don't need to keep it.